models/models/cohort.py

```diff
@@ -52,6 +52,28 @@ class CohortCriteriaInternal(SMBase):
     sg_type: list[str] | None = None
     sample_type: list[str] | None = None
 
+    def to_external(self, project_names: list[str]) -> dict:
```

Comment:
Should probably be returning a `CohortCriteria` object rather than a raw dict?

Reply:
ah, i did it because I was getting this 'Object is not serialisable' error. (thinking...)

Reply:
I probably just need to create a strawberry type for it instead, will do that now